### DISCUSSION TITLE
p-calendar usage in Australia using MS Edge

### DISCUSSION THREAD

**Discussion by thollar** [primefaces/community#151] [PrimeNG] [answered]
Hi,

We have a set of users in Australia (dd/mm/yyyy date format), when using MS Edge browser, if they select the p-calendar control, dates are returned from primeng code correctly if they happen to select a date that is acceptable for both dd/mm/yyyy format and mm/dd/yyyy format.  But, if they happen to select, for example, January 13th, the date returned from PrimeNg is null.  The code works correctly for Chrome users.

Any ideas?

Thanks.

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can clearly identify the issue?

**Comment by thollar** [comment]
If I flip my local time settings to Brisbane AU, I do not experience the issue, unfortunately.  Only our Edge users in Australia are receiving this error.  Can I turn on logging w/i PrimeNG to give provide you with additional information?  If I create a stackblitz example, the issue will not be exposed.  What is the best approach to expose the issue?

**Reply by cetincakiroglu** [reply]
It's a rare edge case and we didn't receive any issue similar to this. Could you please explain "logging w/i PrimeNG to give provide you with additional information" ?

We need to reproduce the issues to identify them clearly and fix them, unfortunately for now we cannot provide a solution.

**Reply by thollar** [reply]
When a user selects a date, the UI displays that date correctly (in dd/mm/yyyy format).  Although, if the date is not a valid mm/dd/yyyy format (not valid Australia format), the date is passed back as null to the model in the app.  I was wondering if you have internal logging that you can turn on that may detect the error.

**Reply by cagataycivici** [reply]
There is no internal logging I'm afraid.

**Reply by ThoSap** [reply, marked as answer]
Hi,

maybe this helps.
You can change your location and locale in Chrome as follows:
https://developer.chrome.com/docs/devtools/settings/locations/

For example:
```
Location name: Sydney
Latitude: -33.8567844
Longitude: 151.2152967
Timezone ID: Australia/Sydney
Locale: en-AU
```
Here you can look up the **canonical** tzdb values for `Australia/*` if you want to try others.
https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
For the coordinates just search the city in Google Maps and copy the coordinates from the URL.

The same approach applies to the Chromium based MS Edge browser.
